tests/runtime/kernel_reg_test.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+import unittest
+
+import torch
+
+from shark_turbine.runtime.op_reg import *
+
+
+class KernelRegTest(unittest.TestCase):
+    def testSimple(self):
+        @CustomOp.register
+        class identity(CustomOp):
+            name = "test_identity"
+            signature = "(Tensor self) -> Tensor"
```

Comment:
Will users have to write these fancy formatted strings for each implementation?

Reply:
Yes, this is just PyTorch syntax for a kernel registration.